scripts/install

```diff
@@ -1,18 +1,29 @@
-#!/bin/sh -e
+#!/usr/bin/env bash
+
+set euxo -pipefail
```

Comment:
What does this do?

Reply:
I'm using it in all my bash scripts since I read this a few years ago (https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail)

can remove it if you want

Reply:
e exits when a command fails
o pipefail set the exit code to the righmost cmd to exit with a non 0 status
with u you're not allowed to use unset variables
x is useful for debug, it print the commands befre executing them

Reply:
and you're right there was a typo it's `-euxo pipefail`
https://explainshell.com/explain?cmd=set+-euxo+pipefail